tests/ssl-opt.sh

```diff
@@ -9242,7 +9239,7 @@ run_test    "DTLS fragmenting: 3d, openssl server, DTLS 1.0" \
             -c "fragmenting handshake message" \
             -C "error"
 
-skip_next_test
+requires_openssl_next
```

Comment:
Adding `requires_openssl_next` doesn't do anything on its own, except for skipping the test. `requires_openssl_next` should only be used when the test case calls `$O_NEXT_CLI` or `$O_NEXT_SRV`. (In fact `ssl-opt.sh` could be improved to detect calls to `$O_NEXT_xxx` and not have to explicitly call `requires_openssl_next`; same for many other `requires_xxx`.)

So now the test case is running with the default OpenSSL version, not the “next” version. And it's passing.

I think that the bug was only with an OpenSSL server, and this test case uses an OpenSSL client against an Mbed TLS client. So it was a mistake to disable it at the time. And now we should fix this by removing `skip_next_test`, not by adding a requirement on openssl_next.

Also applies to development.